chart/skywalking/templates/oap-deployment.yaml

```diff
@@ -122,6 +122,17 @@ spec:
         - containerPort: {{ $value }}
           name: {{ $key }}
         {{- end }}
+
+{{- if eq .Values.oap.env.SW_RECEIVER_ZIPKIN "default" }}
+        - containerPort: 9411
+          name: "zipkin"
+{{- end }}
+
+{{- if eq .Values.oap.env.SW_QUERY_ZIPKIN "default" }}
+        - containerPort: 9412
+          name: "zipkin-ui"
```

Comment:
FYI @jcchavezs Another new Zipkin is being added.

Reply:
@wu-sheng I think it's the original [Zipkin](https://skywalking.apache.org/docs/main/next/en/setup/backend/zipkin-trace/) bounded in Skywalking OAP Server. We just configure the deployment, making it easier to use Zipkin trace in Skywalking on k8s cluster since I find there're some issues about the configuration of Zipkin.

Reply:
Yes, but it was not exposed I think.

Reply:
This is for the sake of convenience, nothing new is added, nothing old was blocked without this changes